### PR TITLE
Add other message types to README

### PR DESCRIPTION
**Pull request by Flova**
Add a list of implemented and coming vision messages.

### CODE REVIEW

**Review by ijnek — APPROVED** [review]
Thanks for this, LGTM. I'll leave the merge for you to do, I don't mind any merge method. (squash, rebase, etc).

**Comment by jaagut** [comment]
Line markings are missing now...^^

**Comment by ijnek** [comment]
> Line markings are missing now...^^

I feel that all the marking should be in a "MarkingArray.msg", rather than separate Arrays for the three types of marking. What do you think?

**Comment by jaagut** [comment]
Yeah, probably. Less spam and I see no problem.

File: soccer_vision_msgs/README.md [hunk 1]
```diff
@@ -5,7 +5,14 @@ This package provides messages for visually detected objects on a soccer field,
 ## Messages (.msg)
 
 * Ball
-
+* BallArray
+* FieldBoundary
+* Goalpost
+* GoalpostArray
```
Comment (ijnek) [inline soccer_vision_msgs/README.md:11]:
```suggestion
* GoalpostArray
* MarkingEllipse
* MarkingIntersection
* MarkingSegment
* MarkingArray
```

**Review by jaagut — APPROVED** [review]

**Pull request merged**